src/wtc-perspective-card.js

```diff
@@ -792,7 +792,7 @@ class ClickablePerspectiveCard extends PerspectiveCard {
    * @listens scroll
    */
 
-  resize(e) {
+  resize(e, force = false) {
     const resize = () => {
```

Comment:
I think creating this `resize` function _inside_ the prototype's resize method might be some unnecessary overhead, since every time we resize the browser, we're creating a new function. Can we move this inner one outside of the class method?

Reply:
Updated, how does that look? I've tested it locally and it's still working fine!

Reply:
Nice 🔥 

Per our off-thread chat, QA team is also good with the change. Good to merge